docs/guidelines.md

```diff
@@ -0,0 +1,87 @@
+# Feedback guidelines
+
+The goal of a language track on Exercism is to give people a way to achieve a
+high level of fluency at a low level of proficiency. We're aiming for fluency
+in the syntax, idioms, and the standard library of the language. You can read
+more about the [goal of exercism here](https://github.com/exercism/docs/blob/master/about/goal-of-exercism.md).
+
+## Definitions
+
+In the following paragraphs, keywords such as **MUST**, **SHOULD**, **COULD**
+are to be interpreted as in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt);
+given that we recognise the following four output states and their restrictions:
+
+- `approve_as_optimal`: **COULD** be a best solution, **SHOULD** be without comment
+- `approve_with_comment`: **COULD** be a best solution, **MUST** be with comment
+- `disapprove_with_comment`: **MUST** be with comment
+- `refer_to_mentor`: **COULD** be with comment
+
+## Conditions and outcome
+
+For feedback generated by automated mentoring:
+
+- it **SHOULD** `disapprove_with_comment` when someone isn't using one of the
+  subjects the exercise is supposed to teach,
+- each `disapprove_with_comment` **MUST** have at least one comment,
+- each `disapprove_with_comment` **SHOULD** steer a student towards an `approve_`,
+- a student **SHOULD NOT** get a `refer_to_mentor` in a newer iteration, if they've
+  had a `disapprove_with_comment` in a previous iteration,
+- a `refer_to_mentor` **COULD** have one or more comments which will be given to
+  the mentor.
+
+There is no minimum or maximum amount of :speech_balloon: comments. However, each
+comment **SHOULD** be aiding the student towards an approvable solution.
+
+## Approvability
+
+Given the above, and to re-iterate that we focus on language fluency:
+
+### Incorrect Naming or Casing
+
+In general, if it's a _language feature_ that will be caught by a compiler or parser
+or based on official rules (which means there is a dependency on correctness in
+tools), you should `disapprove_with_comment`, preferably linking to the official
+rules. If it's a _stylistic_ choice, or there are _competing standards_, do not
+remark at all.
```

Comment:
Suggestion for the last sentence:

If it's a _stylistic_ choice, **and** there are _competing standards_, do not remark at all. **If there's one clear standard, even if not official, consider commenting on it.**

(Marked changes in bold).